styles.css

```diff
@@ -968,27 +1047,27 @@ label {
   }
 
   #contact {
-    margin: 2em 0;
+    margin: 0;
     flex-direction: column;
     flex-wrap: wrap;
     justify-content: space-between;
     align-items: center;
     width: 100%;
     line-height: 1rem;
+    padding: 0;
   }
 
   .contact-all {
-    width: 100%;
+    width: 70%;
     flex-direction: row;
-    justify-content: space-around;
-    margin: 0;
+    justify-content: center;
     padding: 0;
   }
 
   .contact-a {
     padding: 0;
     box-sizing: border-box;
-    width: 50%;
+    width: 40%;
     margin: 0 2em;
   }
 
```

Comment:
Hey dude!

I suggest you check this part of the code, because it's adding a horizontal scroll bar in the tablet version.
Maybe, it's the 2em of margin (line 999 of the styles.css).

![image](https://user-images.githubusercontent.com/96673631/230811741-171acf13-1374-4261-9f05-07a759e905df.png)

Reply:
Thanks for your suggestion bro but I think there was an error when cloning the repository, so I checked the code on github and the changes are there as you can see
![Captura de pantalla (189)](https://user-images.githubusercontent.com/119135868/230897433-dae71081-d5c4-4cc2-8ac8-190e7e54c7e0.png)
![Captura de pantalla (190)](https://user-images.githubusercontent.com/119135868/230897463-396269d4-5ac0-4882-b37d-1848b03f3a73.png)

Reply:
The change has already been made to remove the side scroll
![Captura de pantalla (198)](https://user-images.githubusercontent.com/119135868/230981254-1f03d725-eda7-4274-9e07-22ad167f1c4e.png)